lib/base-compiler.ts

```diff
@@ -2743,6 +2767,26 @@ export class BaseCompiler implements ICompiler {
 
         return output;
     }
+    async processStackUsageOutput(suPath) {
+        const output = parse(fs.readFileSync(suPath, 'utf-8'));
```

Comment:
This is a little bit of a nitpick but can this be done with the async api (i.e. `await fs.readFile`)